sources/packages/web/src/components/common/students/StudentApplications.vue

```diff
@@ -8,76 +8,51 @@
       ></body-header>
     </template>
     <content-group>
-      <DataTable
-        :value="applicationsAndCount.results"
-        :lazy="true"
-        :paginator="true"
-        :rows="DEFAULT_PAGE_LIMIT"
-        :rowsPerPageOptions="PAGINATION_LIST"
-        :totalRecords="applicationsAndCount.count"
-        @page="paginationAndSortEvent($event)"
-        @sort="paginationAndSortEvent($event)"
-        :loading="loading"
+      <toggle-content
+        :toggled="!applicationsAndCount.results?.length"
```

Comment:
Toggle should be enabled based on total count. 

`:toggled="!applicationsAndCount.count"`

Reply:
@dheepak-aot this is not a server-side table right now, so all the records are present on the client side and there is no `count` available. Please let me know if I am missing something.

Reply:
As pointed out by @dheepak-aot API is already accepting the pagination arguments (I missed that), hence can change it.